### PR TITLE
Use regular mutex instead of nsync

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
-remove nsync library

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/process/input/input.go [hunk 1]
```diff
@@ -40,13 +40,17 @@ type Config struct {
...
+	muOverrides          sync.Mutex
+
 	provisionRuntimeInput gqlschema.ProvisionRuntimeInput
 	upgradeRuntimeInput   gqlschema.UpgradeRuntimeInput
 	upgradeShootInput     gqlschema.UpgradeShootInput
-	mutex                 *nsync.NamedMutex
-	overrides             map[string][]*gqlschema.ConfigEntryInput
-	labels                map[string]string
-	globalOverrides       []*gqlschema.ConfigEntryInput
+	//mutex                 *nsync.NamedMutex
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/process/input/input.go:50]: nit: remove this line instead of keeping as a comment?

**Review by wozniakjan — COMMENTED** [review]
just a minor nit with the extra comment, otherwise lgtm

File: components/kyma-environment-broker/internal/process/input/input.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/process/input/input.go:50]: yes, the PR was not ready for review, I forgot to make it draft

**Review by szwedm — APPROVED** [review]
/lgtm

**Pull request merged**